html_app/master_model.js

```diff
@@ -14993,18 +15009,33 @@ var __scb_sample_3 = {
                 }
             },
             add_new_row_instructions: 'add new row instructions',
+            experiment_setup: scb_ex3_text.experiment_setup({}),
             collections:{
 				'30 s': {
-					name: '30 seconds'
+					name: '30 sec'
```

Comment:
Doesn't fit on the line is good enough for me. 

On Feb 27, 2015, at 9:37 AM, annagav <notifications@github.com> wrote:

> In html_app/master_model.js:
> 
> >              collections:{
> >  				'30 s': {
> > -					name: '30 seconds'
> > +					name: '30 sec'
> I think it depends on how Alison and Lourdes decide. I thought full spelling is too long and doesn't fit on the line.
> 
> —
> Reply to this email directly or view it on GitHub.
>